README.md

```diff
@@ -965,15 +965,21 @@ not persisted between runs.
 
 ## Parser cache
 
-A cache of parsed Puppet modules and manifests can be created to skip the use
+One or more caches of parsed Puppet modules and manifests can be created to skip the use
```

Comment:
@ekohl does [this](https://github.com/theforeman/kafo/pull/169) make sense? @komidore64 could you please take a look and verify I don't lie? :-)